src/app/modules/funds/funds.service.ts

```diff
@@ -1,17 +1,30 @@
-import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
-import { Observable } from 'rxjs';
-import { URL } from '../../core/constants/api';
-import { Fund } from '../../shared/models/fund';
+import {Injectable} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {Observable, Subject} from 'rxjs';
+
+import {URL} from '../../core/constants/api';
+import {Fund} from '../../shared/models/fund';
 
 @Injectable({
   providedIn: 'root'
 })
+
 export class FundsService {
-  constructor( private http: HttpClient ) {
-  }
+  private subject = new Subject<any>();
 
-  getFunds(): Observable<Fund[]> {
-    return this.http.get<Fund[]>(`${URL}/funds`);
+  constructor(private http: HttpClient) {
   }
+
+  getFunds = (): Observable<Fund[]> => this.http.get<Fund[]>(`${URL}/funds`);
+
+  create = (fund: Fund): void => {
+    this.http.post(`${URL}/funds/create`, fund)
```

Comment:
It would be better to allocate `${URL}/funds` into separate constant and use it in all needed places.

Reply:
done